ensembl_anno.py

```diff
@@ -5055,6 +4795,7 @@ def coallate_results(main_output_dir):
     # final genes in cases where transcriptomic data are missing or fragmented
     if run_busco:
         logger.info("Running GenBlast of BUSCO proteins")
+        logger.info("run_busco genome file %s", masked_genome_file)
```

Comment:
Let's make a note to re-name this part of the code to run_genblast_orthodb or something like that - the busco name is really confusing :-/